docs/source/en/training/text2image.mdx

```diff
@@ -33,32 +28,43 @@ pip install git+https://github.com/huggingface/diffusers.git
 pip install -U -r requirements.txt
 ```
 
-And initialize an [🤗Accelerate](https://github.com/huggingface/accelerate/) environment with:
+And initialize an [🤗 Accelerate](https://github.com/huggingface/accelerate/) environment with:
 
 ```bash
 accelerate config
 ```
 
-You need to accept the model license before downloading or using the weights. In this example we'll use model version `v1-4`, so you'll need to visit [its card](https://huggingface.co/CompVis/stable-diffusion-v1-4), read the license and tick the checkbox if you agree. 
+You'll need to accept the model license before downloading or using the weights. In this guide, you'll use the `CompVis/stable-diffusion-v1-4` model, so you'll need to visit the [model card](https://huggingface.co/CompVis/stable-diffusion-v1-4), read the license, and tick the checkbox if you agree. 
 
-You have to be a registered user in 🤗 Hugging Face Hub, and you'll also need to use an access token for the code to work. For more information on access tokens, please refer to [this section of the documentation](https://huggingface.co/docs/hub/security-tokens).
+You have to have a 🤗 Hugging Face account, and you'll also need an access token. For more information on access tokens, please refer to the [User access tokens](https://huggingface.co/docs/hub/security-tokens) guide.
 
-Run the following command to authenticate your token
+Run the following command to login and authenticate your token:
 
 ```bash
 huggingface-cli login
 ```
 
 If you have already cloned the repo, then you won't need to go through these steps. Instead, you can pass the path to your local checkout to the training script and it will be loaded from there.
 
-### Hardware Requirements for Fine-tuning
+## Hardware requirements
+
+Using `gradient_checkpointing` and `mixed_precision`, it should be possible to finetune the model on a single 24GB GPU. For higher `batch_size`'s and faster training, it's better to use GPUs with more than 30GB of GPU memory. You can also use JAX/Flax for fine-tuning on TPUs or GPUs, which will be covered [below](#flax-jax-finetuning).
 
-Using `gradient_checkpointing` and `mixed_precision` it should be possible to fine tune the model on a single 24GB GPU. For higher `batch_size` and faster training it's better to use GPUs with more than 30GB of GPU memory. You can also use JAX / Flax for fine-tuning on TPUs or GPUs, see [below](#flax-jax-finetuning) for details.
+## xFormers
 
-### Fine-tuning Example
+You can enable memory efficient attention by [installing xFormers](./optimization/xformers) and passing the `--enable_xformers_memory_efficient_attention` flag to the training script.
 
-The following script will launch a fine-tuning run using [Justin Pinkneys' captioned Pokemon dataset](https://huggingface.co/datasets/lambdalabs/pokemon-blip-captions), available in Hugging Face Hub.
+xFormers is not available for Flax.
 
+## LoRA
```

Comment:
Why are we showing LoRA here? LoRA is a very specific case of text-to-image fine-tuning. We should just show normal text-to-image fine-tuning here